expression/builtin_math_vec_test.go

```diff
@@ -23,7 +23,10 @@ import (
 )
 
 var vecBuiltinMathCases = map[string][]vecExprBenchCase{
-	ast.Sign: {},
+	ast.Sign: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}, geners: []dataGenerator{nil, nil}},
```

Comment:
It seems we don't need this case, which is duplicated with the one above it.

Reply:
That's a mistake. It's solved.